crates/agglayer-storage/src/stores/state/mod.rs

```diff
@@ -485,11 +494,21 @@ impl StateReader for StateStore {
         &self,
         network_id: NetworkId,
     ) -> Result<Option<LocalNetworkStateData>, Error> {
-        match (
-            self.read_local_exit_tree(network_id)?,
-            self.read_smt::<BalanceTreePerNetworkColumn, LOCAL_BALANCE_TREE_DEPTH>(network_id)?,
-            self.read_smt::<NullifierTreePerNetworkColumn, NULLIFIER_TREE_DEPTH>(network_id)?,
-        ) {
+        debug!("Reading local network state for network_id: {}", network_id);
```

Comment:
I don't think we need to keep all of those debug right?

Reply:
Removed, I kept only the 1st log